mautic/mautic
# Allow themes to be used with multiple builders

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check the list of available themes. Among others "Blank" should be available. These themes should **not** be available:
  - Brienz
  - Paprika
3. Enable grapesjs
4. The following 2 themes should now be available in addition to "Blank". 
  - Brienz
  - Paprika

fixes: https://github.com/mautic/plugin-grapesjs-builder/issues/51
replaces: https://github.com/mautic/mautic/pull/9841
